src/lib/components/Toast.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  import { fade } from "svelte/transition";
+  import { toasts } from "$lib/stores/toast";
+</script>
+
+{#if $toasts}
+  <div class="toast">
+    {#each $toasts as toast (toast.id)}
+      <div role="alert" class="alert alert-{toast.type}" out:fade={{ duration: 750 }}>
```

Comment:
Maybe offset the Toasts a bit from the edge of the screen? i.e add the class `right-4`